source/includes/elastic-agents/_030-get-profile-metadata.md.erb

```diff
@@ -0,0 +1,33 @@
+## Get Profile Metadata
+
+This is a message that the plugin should, to allow users to configure profiles from the Elastic Profiles View in GoCD.
+
+<p class='request-name-heading'>Request name</p>
+
+`go.cd.elastic-agent.get-profile-metadata`
+
+<p class='request-body-heading'>Request body</p>
+
+The server will not provide a request body.
+
+<p class='response-code-heading'>Response code</p>
+
+The plugin is expected to return status `200` if it can understand the request.
+
+<p class='response-body-heading'>Response Body</p>
+
+> An example response body for a docker plugin
+
+```json
+[
+  {
+    "key": "Image",
+    "metadata": {
+      "required": true,
+      "secure": false
```

Comment:
The docker elastic agents require only these two attributes. Is there any need for the other attributes to be shown like ```display_name``` and ```display_order```? Should we mention that here?